<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formModifierPersonnage</class>
 <widget class="QDialog" name="formModifierPersonnage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>791</width>
     <height>431</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="FeuillePersonnage">
    <widget class="QPushButton" name="InventaireButton">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>290</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Inventaire</string>
     </property>
    </widget>
    <widget class="QLabel" name="Attribut_1Label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>162</y>
       <width>83</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Attribut 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>231</x>
       <y>10</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Feuille de Personnage :</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>389</x>
       <y>66</y>
       <width>62</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Race :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>117</y>
       <width>759</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>______________________________________________________________________________</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QListWidget" name="CompetenceList">
     <property name="geometry">
      <rect>
       <x>536</x>
       <y>162</y>
       <width>233</width>
       <height>73</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="PVmaxEdit">
     <property name="geometry">
      <rect>
       <x>329</x>
       <y>136</y>
       <width>86</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Attribut_4Edit">
     <property name="geometry">
      <rect>
       <x>421</x>
       <y>162</y>
       <width>109</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Attribut_4Label">
     <property name="geometry">
      <rect>
       <x>307</x>
       <y>162</y>
       <width>108</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Attribut 4</string>
     </property>
    </widget>
    <widget class="QLabel" name="SanteLabel">
     <property name="geometry">
      <rect>
       <x>91</x>
       <y>136</y>
       <width>94</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>PV :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>307</x>
       <y>136</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
    <widget class="QLabel" name="ImagePerosnnageLabel">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>18</y>
       <width>111</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="ClasseEdit">
     <property name="geometry">
      <rect>
       <x>457</x>
       <y>92</y>
       <width>155</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Attribut_2Edit">
     <property name="geometry">
      <rect>
       <x>99</x>
       <y>188</y>
       <width>86</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>142</x>
       <y>92</y>
       <width>43</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PrenomPersonnageEdit">
     <property name="geometry">
      <rect>
       <x>191</x>
       <y>66</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Attribut_2Label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>188</y>
       <width>75</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Attribut 2 </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>389</x>
       <y>92</y>
       <width>62</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Classe :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PVactuelEdit">
     <property name="geometry">
      <rect>
       <x>191</x>
       <y>136</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Attribut_3Edit">
     <property name="geometry">
      <rect>
       <x>99</x>
       <y>214</y>
       <width>86</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>411</x>
       <y>40</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Joueur :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>145</x>
       <y>238</y>
       <width>155</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Descriptif :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>635</x>
       <y>68</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Niveau :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>641</x>
       <y>96</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Sexe :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>536</x>
       <y>136</y>
       <width>233</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Comp√©tences :</string>
     </property>
    </widget>
    <widget class="QLabel" name="Attribut_3Label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>214</y>
       <width>75</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Attribut 3</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="DescriptifEdit">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>268</y>
       <width>678</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="RaceEdit">
     <property name="geometry">
      <rect>
       <x>457</x>
       <y>66</y>
       <width>155</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="retourButton">
     <property name="geometry">
      <rect>
       <x>705</x>
       <y>408</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Retour</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="NomEdit">
     <property name="geometry">
      <rect>
       <x>191</x>
       <y>92</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Attribut_6Edit">
     <property name="geometry">
      <rect>
       <x>421</x>
       <y>214</y>
       <width>109</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Attribut_5Edit">
     <property name="geometry">
      <rect>
       <x>421</x>
       <y>188</y>
       <width>109</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="AgeEdit">
     <property name="geometry">
      <rect>
       <x>687</x>
       <y>46</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Attribut_1Edit">
     <property name="geometry">
      <rect>
       <x>99</x>
       <y>162</y>
       <width>86</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>142</x>
       <y>66</y>
       <width>43</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Pr√©nom :</string>
     </property>
    </widget>
    <widget class="QLabel" name="Attribut_6Label">
     <property name="geometry">
      <rect>
       <x>307</x>
       <y>214</y>
       <width>108</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Attribut 6</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>641</x>
       <y>46</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Age :</string>
     </property>
    </widget>
    <widget class="QLabel" name="Attribut_5Label">
     <property name="geometry">
      <rect>
       <x>307</x>
       <y>188</y>
       <width>108</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Attribut 5</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="SexeEdit">
     <property name="geometry">
      <rect>
       <x>687</x>
       <y>98</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="NiveauEdit">
     <property name="geometry">
      <rect>
       <x>687</x>
       <y>72</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Inventaire">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>40</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Autres :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Encombrement :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ArgentLineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="ObjetTable">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>90</y>
       <width>291</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="EncombrementValeurEdit">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>240</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Poids</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>D√©gats</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>70</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Poids</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="EncombrementSeuilEdit">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>70</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Usure</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>240</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Usure</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Armes :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QTableView" name="ArmesTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>451</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>20</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>240</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Protection</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>70</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Qt√©</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Armures :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>70</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Usage</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Argent :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>70</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Valeur</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>70</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>70</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Poids</string>
     </property>
    </widget>
    <widget class="QTableView" name="ArmureTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>451</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_33">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>10</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Inventaire</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="RetourButton">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>390</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Retour</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
